week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -57,10 +71,39 @@ function exerciseTwo(shopping) {
 
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
+
+
+let url1 = "https://image.ebooks.com/previews/095/095767/095767915/095767915-hq-168-80.jpg"
+let url2 = "https://images-na.ssl-images-amazon.com/images/I/41m1rQjm5tL._SX322_BO1,204,203,200_.jpg"
+let url3 = "https://wordery.com/jackets/892d2e10/m/the-pragmatic-programmer-david-thomas-9780135957059.jpg"
+let urls = [url1, url2, url3]
+
 function exerciseThree(books) {
-  //Write your code in here
+  let list = document.createElement("ul");
+  let content = document.getElementById("content");
+  content.appendChild(list);
+  // books[0].url = url1;
+  // books[1].url = url2;
+  // books[2].url = url3;
+  books.map((book, i) => {
+    let listItem = document.createElement("li");
+    list.appendChild(listItem);
+    let parEl = document.createElement("p");
+    listItem.appendChild(parEl);
+    parEl.innerHTML = `${book.title} - ${book.author}`;
```

Comment:
Considering `${book.title} - ${book.author}` is just a string, could you be using a different DOM method other than `.innerHTML`?

Reply:
Yes, .innerText is better